### DISCUSSION TITLE
Pull request creation broken

### DISCUSSION THREAD

**Discussion by michelle-chiang** [community/community#41310] [Pull Requests]
### Select Topic Area

Bug

### Body

Right after clicking "Create a Pull Request", the normal PR view is shown. However, after refreshing:
- The url redirects to `github.com/.../issues/<pull-request-number>`
- The new PR doesn't show up under the "Pull requests" repository tab

**Comment by anonrig** [comment]
Conversation resolution is broken too. The corresponding endpoint returns: 

```
{
  "errors": "User is not authorized to resolve the conversation"
}
```

**Comment by anonrig** [comment]
Broken too: Updating a fork (force-push), does not update pull request.
